src/UnitySampleProject/ProjectSettings/ProjectSettings.asset

```diff
@@ -3,7 +3,7 @@
 --- !u!129 &1
 PlayerSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 24
+  serializedVersion: 23
```

Comment:
@robGG1997 Did you use an older Unity Editor and if so for what reason ? A lot of packages in package-lock.json also have a lower package values 🤔

Reply:
@kantagara yes I'm using a slightly lower version, .3.15 instead of 3.22 but I've reverted unwanted changes included in merge from main, this should be outdated